jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-compute-template.yaml

```diff
@@ -24,3 +24,4 @@
             scenario=cloud{version}-4nodes-compute-ha.yml
             want_node_aliases=controller=2,compute=2
             label={label}
+            storage_method=none
```

Comment:
the job doesn't know `want_ceph` anymore, only mkcloud still knows it. so `storage_method` in the ci would set it